wp-content/themes/engage/src/Models/ResearchArticle.php

```diff
@@ -28,6 +29,13 @@ public function getSummary() {
     	return $this->summary;
     }
 
+		public function getVideo() {
+			if($this->video === false) {
+    		$this->video = get_field('video_here');
```

Comment:
Could you rename this to be more explicit about what it is? Like, do I put a URL to a video? Or an embed code? Or...? This will help make things more understandable when we're skimming through.